static/src/javascripts/projects/commercial/modules/dfp/load-advert.ts

```diff
@@ -76,7 +76,12 @@ export const refreshAdvert = (advert: Advert): void => {
 				// force the slot sizes to be the same as advert.size (current)
 				// only when advert.size is an array (forget 'fluid' and other specials)
 				if (Array.isArray(advert.size)) {
-					advert.slot.defineSizeMapping([[[0, 0], [advert.size]]]);
+					const mapping = window.googletag.sizeMapping();
+					mapping.addSize(
+						[0, 0],
+						advert.size as googletag.GeneralSize,
+					);
+					advert.slot.defineSizeMapping(mapping.build());
```

Comment:
Is this just a preference to use the mapping builder rather than just an inline call with all the params?

Reply:
I was really struggling to get the types to behave when just passing as an params, I could take another crack at it, it is definitely a bit unnecessarily verbose.